content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,90 @@
+## Installation
+
+- **Added** a command line [Istio operator](docs/setup/install/operator/) to install and manage the Istio control plane.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly classify ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to `MAGLEV` when using the Redis proxy.
+- **Improved** traffic management to direct traffic to the same region and zone.
+- **Improved** Pilot performance. The reduction in CPU utilization could be up to 10 times depending on deployment.
+- **Improved** service entries to allow for the same host name in different namespaces.
+- **Improved** the Sidecar API to customize the `OutboundTrafficPolicy` policy.
```

Comment:
can we add a link here?  https://preliminary.istio.io/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy

Reply:
```suggestion
- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
```

Reply:
approved.